Helper/DeviceView.php

```diff
@@ -66,11 +63,17 @@ class DeviceView
     protected $switchParam = self::SWITCH_PARAM_DEFAULT;
 
     /**
+     * @var array
+     */
+    protected $configRedirect;
+
+    /**
      * Constructor
      *
-     * @param \Symfony\Component\DependencyInjection\Container $serviceContainer
+     * @param RequestStack $requestStack
+     * @param array        $configRedirect
      */
-    public function __construct(RequestStack $requestStack = null)
+    public function __construct(RequestStack $requestStack = null, $configRedirect = null)
```

Comment:
Now that you can inject config values into the constructor it IMO seems logical to be able to configure these values through the semantic config of the bundle.